modules/carto/src/index.ts

```diff
@@ -2,7 +2,8 @@ export {getDefaultCredentials, setDefaultCredentials} from './config';
 export {default as CartoLayer} from './layers/carto-layer';
 export {default as _CartoTileLayer} from './layers/carto-tile-layer';
 export {default as _H3TileLayer} from './layers/h3-tile-layer';
-export {default as _QuadkeyTileLayer} from './layers/quadkey-tile-layer';
+export {default as _QuadbinLayer} from './layers/quadbin-layer';
```

Comment:
Do we really need to export _QuadbinLayer?

Reply:
@padawannn needs it in Builder I think

Reply:
@alasarr I need it for builder as the rest of the layer exported

Reply:
I mean @padawannn needs _QuadbinTileLayer not _QuadbinLayer

Reply:
Sorry I really need the `_QuadbinTileLayer`